rebar.config

```diff
@@ -1,37 +1,69 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2010-2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
 {src_dirs, ["./priv/tracers", "./src"]}.
+
 {cover_enabled, false}.
+
 {edoc_opts, [{preprocess, true}]}.
-{erl_opts, [warnings_as_errors,
-            {parse_transform, lager_transform},
-            {src_dirs, ["src", "priv/tracers"]},
-            {platform_define, "^[0-9]+", namespaced_types},
-            {platform_define, "^[0-9]+", set_env_options},
-            {platform_define, "^R15", "old_hash"},
-            {d, 'TEST_FS2_BACKEND_IN_RIAK_KV'}]}.
+
+{erl_opts, [
+    debug_info,
+    warnings_as_errors,
+    {parse_transform, lager_transform},
+    {src_dirs, ["src", "priv/tracers"]},
+    {platform_define, "^[0-9]+", namespaced_types},
+    {platform_define, "^[0-9]+", set_env_options},
+    {platform_define, "^R15", "old_hash"},
+    {d, 'TEST_FS2_BACKEND_IN_RIAK_KV'}
+]}.
 
 {eunit_opts, [
-     no_tty,  %% This turns off the default output, MUST HAVE
-     {report, {eunit_progress, [colored, profile]}} %% Use `profile' to see test timing information
-     %% Uses the progress formatter with ANSI-colored output
-     ]}.
+    %% This turns off the default output, MUST HAVE
+    no_tty,
+    %% Use the progress formatter with ANSI-colored output
+    %% Use `profile' to see test timing information
+    {report, {eunit_progress, [colored, profile]}}
+]}.
 
 {xref_checks, []}.
-%% XXX yz_kv is here becase Ryan has not yet made a generic hook interface for object modification
-%% XXX yz_stat is here for similar reasons -- we do not yet support dynamic stat hooks
-{xref_queries, [{"(XC - UC) || (XU - X - B - \"(cluster_info|dtrace|yz_kv|yz_stat)\" : Mod)", []}]}.
+{xref_queries, [
+    %% yz_kv because we don't have a generic hook interface for object modification
+    %% yz_stat for similar reasons -- we do not yet support dynamic stat hooks
+    {"(XC - UC) || (XU - X - B - \"(cluster_info|dtrace|yz_kv|yz_stat)\" : Mod)", []}
+]}.
 
 {erl_first_files, [
-                   "src/riak_kv_backend.erl"
-                  ]}.
+    "src/riak_kv_backend.erl"
+]}.
 
 {deps, [
```

Comment:
I definitely agree that we should be moving away from so many tags in our dev branches, but I don't know if I like that these changes are being snuck into an unrelated pull request.

Some of these are relatively innocuous; it looks like sidejob and bitcask are just going to end up pointing to the same commit as they are now (for the time being anyway).

The sext change makes me a little more nervous - any changes to the way stuff gets encoded could require special care and attention, so this is one case where I might be somewhat inclined to leave it as a tag. I guess this is probably okay in this case anyway since we're pointing at our own forked copy of sext, so at least we control when it's updated...

Oh, actually though, I just determined that we shouldn't be updating sext here because it tries to pull in a new version of edown. This will cause a version conflict with the "setup" and "parse_trans" repos, both of which still specifically request the 0.5 tag. We could try updating things on all three repos at once, but I think we should do that as a separate series of PRs instead of trying to tack it onto this effort.

It looks like we also get some new code pulled in for eper and eunit_formatters, but at least those are only debug/test tools, so I'm a little less worried about that. Did you test it with the new versions of those deps by any chance though, just to make sure nothing big breaks?